### PR TITLE
Add TypeScript demos

### PR DESCRIPTION
**Pull request by pepone**
New TypeScript demos for Node.js and Browser

### CODE REVIEW

File: typescript/nodejs/README.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+### Prerequisites
+
+The JavaScript sample programs require an installation of [Node.js][3].
+
+### Building the Demos
+
+#### Building the demos with the NPM packages
+
+With Node.js in your PATH, go to the [typescript/nodejs directory](../typescript/nodejs)
```
Comment (externl) [inline typescript/nodejs/README.md]: Is this link correct?

Comment (pepone) [reply typescript/nodejs/README.md]: No that is bogus, it just the current directory of the README, maybe we should just remove it.

Comment (externl) [reply typescript/nodejs/README.md]: 👍

File: typescript/browser/README.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+- [Chat](./Chat) is a JavaScript client for the ZeroC [Chat Demo][1].
+
+- [Glacier2](./Glacier2) is a simpler chat client that uses the Glacier2 service.
+
+- [Ice](./Ice) has several examples that illustrate fundamental Ice concepts.
+
+- [Manual](./Manual) contains complete examples for some of the code snippets
+in the [Ice manual][2].
+
+Refer to the [C++11 demos](../cpp11) for more examples that use Ice services
```
Comment (externl) [inline typescript/browser/README.md]: This link also looks bogus.

File: typescript/browser/README.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+### Prerequisites
+
+The TypeScript sample programs require an installation of [Node.js][3].
+
+### Building the Demos
+
+#### Building the demos with the NPM packages
+
+With Node.js in your PATH, go to the [typescript/browser directory](../typescript/browser)
```
Comment (externl) [inline typescript/browser/README.md]: As does this one.

**Comment by pepone** [comment]
Merged 19176402c084a26b54a53e80070f7d2661477ed8

**Pull request closed without merging**